csv/src/main/java/com/fasterxml/jackson/dataformat/csv/CsvSchema.java

```diff
@@ -1483,6 +1483,22 @@ public String getColumnDesc()
         sb.append(']');
         return sb.toString();
     }
+    /**
+     * Method for ignoring fields for CsvBuilder
+     * @param ignoreProperties Array of column names to be ignored by the csv builder
+     * @return A newly built {@link CsvSchema} with the ignored properties
+     */
+    public CsvSchema ignoreProperty(String[] ignoreProperties) {
```

Comment:
From naming perspective it should probably be "ignoreProperties".

Also, would perhaps make sense to take `String...` instead of `String[]` for convenience.